controllers/providers/pg/applier.go

```diff
@@ -308,16 +317,7 @@ func defaultSpec(db *everestv1alpha1.DatabaseCluster) pgv2.PerconaPGClusterSpec
 				},
 			},
 		},
-		PMM: &pgv2.PMMSpec{
-			Enabled: false,
-			Resources: corev1.ResourceRequirements{
-				Limits: corev1.ResourceList{
-					corev1.ResourceMemory: resource.MustParse("300M"),
-					corev1.ResourceCPU:    resource.MustParse("500m"),
-				},
-			},
-			Secret: fmt.Sprintf("%s%s-pmm", common.EverestSecretsPrefix, db.GetName()),
-		},
+		PMM: nil,
```

Comment:
Am I understand it correctly that the idea of this refactoring is to unify the `defaultSpec()` signature among the operators? If so, then perhaps it would be more predictable to keep the PMM config inside the default spec as the other operators do, but define only the `Secret` outside. That would make the difference between operators more prominent. 
Just a suggestion, feel free to resolve it if not addressed

Reply:
Actually that was not the reason. The PGO now enforces that one of the fields in this PMMSpec struct is mandatory and we weren't setting it if the monitoring was turned off. In order to fix it, I decided to remove the struct altogether if monitoring is disabled.